langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -104,11 +137,9 @@ public isolated function filter(Type[] arr, @isolatedParam function(Type val) re
 # The combining function takes the combined value so far and a member of the array,
 # and returns a new combined value.
 #
-# For example
-# ```
-# reduce([1, 2, 3], function (int total, int n) returns int { return total + n; }, 0)
+# ```ballerina
+# [1, 2, 3].reduce(isolated function (int total, int next) returns int => total + next, 0) ⇒ 6
```

Comment:
`total` or `accTotal` (I'm referring to accumulated total as `accTotal`)?

Reply:
Doesn't total indicate that? Why do we need to add accumulated?

Reply:
Doesn't the user get a confusing idea? (total and accumulated total are two different things, no?

Please check this too: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

BTW If we want to focus on showing minimal effort, I'm okay using `total`.

Reply:
I'm not sure that `total` and `accumulatedTotal` are two different things, depends on the context, right? Here we are "updating" `total` with the accumulated value so far.

> Please check this too: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Doesn't this use `accumulator`? I think `total` OR `accumulator` make sense, but `accumulatedTotal` doesn't add much value compared to `total`.

Reply:
hmm... Yeah. `total` could be better